builds/e2e/nested-e2e.yaml

```diff
@@ -17,18 +17,20 @@ stages:
 - stage: SetupVM
   jobs:
   - template: templates/nested-parent-vm-setup.yaml
+    parameters:
+      transportType: mqtt
   - job:  SetupVM_and_RunTest_level3
     dependsOn:
-      - SetupVM_level5
```

Comment:
@nyanzebra I think something is not right. Do these changes belong to the PR?

Reply:
oh, I just saw your comment above...

Reply:
Should be fixed now I hope